src/jepsen/dqlite.clj

```diff
@@ -78,6 +78,7 @@
                        :nodes  (:nodes opts)
                        :partition {:targets (:partition-targets opts)}
                        :pause     {:targets [nil :one :primaries :majority :all]}
+                       :stop      {:targets [nil :one :primaries :majority :all]}
```

Comment:
You're right, `nil` is redundant.
Used this as an opportunity to add `--node-targets` opt:
```clj
{:stop {:targets (:node-targets opts)}}
```

This will also support targeted testing:
```
lein run test
  --workload bank
  --nemesis disk
  --node-targets primaries 
```